src/shared/report_op.c

```diff
@@ -385,7 +385,9 @@ void os_ReportdStart(report_filter *r_filter)
     int alerts_filtered = 0;
     char *first_alert = NULL;
     char *last_alert = NULL;
-    alert_data **data_to_clean = NULL;
+    char *msg_show_alerts = NULL;
+    int size_max_msg = 8192;
```

Comment:
This integer will be used in the function `snprintf` to prepare the out, of `showalerts`. I have used `8192` so that the first alert, has enough capacity and the function `snprintf` not limit the characters of the alert. In addition, this variable, because of the changes I'm preparing, is going to move to another part of the code and I am going to use the constant `OS_SIZE_8192` instead of the integer `8192`.